scripts/tapir/upgrade_coordinator.py

```diff
@@ -19,14 +19,17 @@ def main():
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
     instances = contracts_from_registry(filepath=ARTIFACTS_DIR / "tapir.json", chain_id=80002)
 
-    implementation = deployer.deploy(project.Coordinator)
+    # implementation = deployer.deploy(project.Coordinator)
```

Comment:
Is this line being commented out intentional? Don't we need to deploy the implementation contract first?

Reply:
It's intentional - the implementation gets deployed as part of the `deployer.upgrade` call - https://github.com/derekpierre/nucypher-contracts/blob/tapir-new-abi/deployment/params.py#L635.

That implementation line was done before and can be used with `deployer.upgradeTo()` which was something we did in the past but is no longer applicable hence commented out. You'll see the same thing in the `lynx/upgrade_coordinator.py` file which David used for upgrading the lynx Coordinator.